Authors:  Cecil Curry <leycec@gmail.com>
Upstream: no
Synopsis: Allow non-versioned "/usr/${LIBDIR}"/libboost*.so filenames.
--- old/development/scons/scons_utils.py	2008-07-26 10:45:38.000000000 +0000
+++ new/development/scons/scons_utils.py	2010-05-03 17:45:14.976305815 +0000
@@ -311,17 +311,8 @@
                 conf.Log("Find boost libraries: %s\n" % files)
                 # runtime code includes s,g,y,d,p,n, where we should look for
                 # d,g,y for debug, s,p,n for release
-                lib_files = []
-                if isDebug:
-                    for ver in versions:
-                        lib_files += filter(lambda x: re.search('%sboost_%s-\w+-mt-[^spn]+-%s%s' % (libprefix, lib, ver, libsuffix), x), files)
-                else:
-                    for ver in versions:
-                        lib_files += filter(lambda x: re.search('%sboost_%s-\w+-mt-([^dgy]+-)*%s%s' % (libprefix, lib, ver, libsuffix), x), files)
-                if len(lib_files) == 0:
-                    # use alternative libraries
-                    for ver in versions:
-                        lib_files += filter(lambda x: re.search('%sboost_%s-[\w-]+%s%s' % (libprefix, lib, ver, libsuffix), x), files)
+                lib_files = files
+
                 if len(lib_files) > 0:
                     # get xxx-gcc-1_33_1 from /usr/local/lib/libboost_xxx-gcc-1_33_1.a
                     name = lib_files[0].split(os.sep)[-1][len(libprefix):]
